tests/commands/test_unset.py

```diff
@@ -8,32 +8,31 @@
          'lap top_ap ple_mac book pro']
 
 directories = ['.',
-               'simple',
                's p a c e s',
-               's p a/c e s',
                'r/e/c/u/r/s/i/v/e',
                'overlap/one',
                'overlap/two',
-               'very/very/very/deep']
+               'very/very/very/deep',
+               ]
 
 assets = [f"{d}/{f}.{i}" for f in files for i, d in enumerate(directories)]
 
+content_dict = {"key": "value",
+                "second": "input",
+                "another": "field"}
 
-@pytest.mark.repo_files(*assets)
+content_str = "\n".join([f"{elem}: {content_dict.get(elem)}"
```

Comment:
IMO, this is easier to read if it's not built programmatically, and instead as a literal string.

Reply:
For the people of the future, reading through all our PRs and trying to follow our thought process, I want to mention, that we decided against hardcoding strings and other variables after all, so as to enable us in the future to easier update our tests.